bin/git-effort

```diff
@@ -207,11 +214,15 @@ heading
 
 # send paths to effort
 nPaths=${#paths[@]}
+nProcs=${$(procs):-8}
```

Comment:
I got `${$(procs):-8}: bad substitution` error here. Maybe we can move the fallback code to function `procs`?

Reply:
> I got ${$(procs):-8}: bad substitution error here
It seems you're using `sh` instead of `bash`. Could you tell me which shell are you using? 
> Maybe we can move the fallback code to function procs?
Sure.

Reply:
My bash doesn't work with this...
```
 ￥ bash -c 'nProcs=${$(procs):-8}'
bash: ${$(procs):-8}: bad substitution
 ￥ bash --version
GNU bash, version 5.0.17(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2019 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```

Reply:
reproduced and fixed.

Reply:
Before I merged this, since I used `wait -n` which was introduced in bash 4.0, should I change the CONTRIBUTING.md?